### PR TITLE
Adds updates from kdf135 tls and srtp interop testing

### PR DESCRIPTION
**Pull request by elliemdaw**

### CODE REVIEW

File: app/app_main.c [hunk 1]
```diff
@@ -952,7 +952,7 @@ static void enable_kdf (ACVP_CTX *ctx) {
     /*
      * Enable KDF-135
      */
-    rv = acvp_enable_kdf135_tls_cap(ctx, ACVP_KDF135_TLS, &app_kdf135_tls_handler);
```
Comment (bfussell) [inline app/app_main.c]: Are we removing the KDF code from the app in github at some point ?

Comment (elliemdaw) [reply app/app_main.c]: Ahh so I hadn't added any of the new kdf code, but I'll take out the ones that are in there from earlier. Give me a minute :)

Comment (elliemdaw) [reply app/app_main.c]: Actually I think it may be good to leave what we have for reference. I had already taken out our API calls.

Comment (bfussell) [reply app/app_main.c]: OK, got it. registration is good, handler implementation bad.

**Pull request merged**